README.md

```diff
@@ -108,7 +51,7 @@ Available commands:
 
 Each command allows interacting with a product of the Datadog platform. The commands are defined in the [src/commands](/src/commands) folder.
 
-Further documentation for each command can be found in its folder, ie:
+Further documentation for these commands can be found in their respective folders:
```

Comment:
Maybe we could merge the "Available commands" list in the Usage just above with the documentation of each of the commands?

```bash
Usage: datadog-ci <command> <subcommand> [options]
```

Possible values for `<command>`:
- `dsyms` (see [iOS dSYM Files](src/commands/dsyms/))
- `flutter-symbols` (see [Flutter Symbols](src/commands/flutter-symbols/))
- `git-metadata` (see [Git metadata](src/commands/git-metadata))
- `junit` (see [JUnit XML](src/commands/junit))
- `lambda` (see [Lambda](src/commands/lambda))
- `metric` (see [Metric](src/commands/metric))
- `react-native` (see [React Native sourcemaps](src/commands/react-native/))
- `sourcemaps` (see [Browser sourcemaps](src/commands/sourcemaps/))
- `synthetics` (see [Synthetics CI/CD Testing](src/commands/synthetics/))
- `tag` (see [Tag](src/commands/tag))
- `trace` (see [Trace](src/commands/trace))